criterion-measurement/criterion-measurement.cabal

```diff
@@ -0,0 +1,68 @@
+name:                criterion-measurement
+version:             0.1.0.0
+synopsis:            Criterion measurement functionality and associated types
+description:         Measurement-related functionality extracted from Criterion, with minimal dependencies. The rationale for this is to enable alternative analysis front-ends.
+homepage:            https://github.com/bos/criterion
+license:             BSD3
+license-file:        LICENSE
+author:              Bryan O'Sullivan <bos@serpentine.com>
+maintainer:          Marco Zocca <zocca.marco gmail>, Ryan Scott
+copyright:           2009-2016 Bryan O'Sullivan and others
+category:            Development, Performance, Testing, Benchmarking
+build-type:          Simple
+extra-source-files:  README.md
+cabal-version:       >=1.10
+tested-with:
+  GHC==7.4.2,
+  GHC==7.6.3,
+  GHC==7.8.4,
+  GHC==7.10.3,
+  GHC==8.0.2,
+  GHC==8.2.2,
+  GHC==8.4.1
+
+flag fast
+  description: compile without optimizations
+  default: False
+  manual: True
+
+library
+  hs-source-dirs:      src
+  exposed-modules:     Criterion.Measurement
+                       Criterion.Types
+  other-modules:       Criterion.Types.Internal
+  build-depends:       aeson >= 0.8
+                     , base >= 4.7 && < 5
+                     , base-compat >= 0.9
+                     , binary >= 0.5.1.0
+                     , containers
+                     , deepseq >= 1.1.0.0
+                     , vector >= 0.7.1
+  if impl(ghc < 7.6)
+    build-depends:
+      ghc-prim
+
+  default-language: Haskell2010
+  ghc-options: -Wall -funbox-strict-fields
+  if impl(ghc >= 6.8)
+    ghc-options: -fwarn-tabs
+  if flag(fast)
+    ghc-options: -O0
+  else
+    ghc-options: -O2
+
+
+  c-sources: cbits/cycles.c
```

Comment:
Since the `cbits` stuff lives in `criterion-measurement` now, we should be able to remove the corresponding `cbits` parts from `criterion.cabal`, yes? (Also, we should be able to remove the top-level `cbits` directory, since they should now live solely in `criterion-measurement/cbits`, IIUC.)